examples/agent_multihop_qa.py

```diff
@@ -109,5 +109,4 @@
 ]
 
 for question in hotpot_questions:
-    result = agent.run(query=question)
```

Comment:
I would prefer the print statement in the example. The other examples have one too and otherwise the user won't see any result if they run the example.
```python
result = agent.run(query=question)
print(result)
```